Chinese/Interstellar/Interstellar.md

```diff
@@ -0,0 +1,7 @@
+爆炸声打破了死寂，孙悟空感到身体在灼烧，他挣扎着睁开双眼：
+
+周围是大量的仪表盘和屏幕，眼前是一望无际的黑暗。安妮海瑟薇的飞船在他身后爆炸，把他的太空舱推入了黑洞。
+
+他来不及细想上一幕自己身在何方，也无暇思考当前发生的一切。他好像进入了真理之门，五百年来发生的一幕幕都以2的5次方倍速在他面前重现，悟空顿觉脑袋胀裂，痛不欲生。当画面来到最后一幕：五指山下，悟空蓬头垢面，三藏袈裟翩翩，正站在他面前，对他说了一句：“荆轲刺秦王”。
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.